h/views.py

```diff
@@ -1,16 +1,18 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 import logging
-import re
 
 from pyramid import httpexceptions
 from pyramid.events import ContextFound
 from pyramid.view import forbidden_view_config, notfound_view_config
 from pyramid.view import view_config
+import pyramid.response
 
 from . import session
 from .models import Annotation
 from .resources import Application, Stream
+import h.api_client
```

Comment:
Fine for me
On Mar 19, 2015 6:40 PM, "Nick Stenning" <notifications@github.com> wrote:

> In h/views.py
> <https://github.com/hypothesis/h/pull/2029#discussion_r26805191>:
>
> >
> >  from . import session
> >  from .models import Annotation
> >  from .resources import Application, Stream
> > +import h.api_client
>
> +1 to relative imports, but I'd prefer them one to a line, especially if
> they're not related, as it greatly improves diff readability.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hypothesis/h/pull/2029/files#r26805191>.
>